#! /usr/bin/env python

PACKAGE = "robotx_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("motor_param_forward", double_t, 0, "motor param forward", 700, 0, 2500)
gen.add("motor_param_angular", double_t, 0, "motor param angular ", 300, 0, 500)

exit(gen.generate(PACKAGE, "robotx_control", "StartMotor"))
